docs/lib/datastore/fragments/js/sync/50-selectiveSync.md

```diff
@@ -1,8 +1,11 @@
-## Selective Sync
+## Selectively syncing a subset of your data
 
-By default, DataStore persists the entire contents of your data source. (The max number of records is configurable here: https://docs.amplify.aws/lib/datastore/conflict/q/platform/js#example).
+By default, DataStore persists the entire contents of your cloud data source to the local store. 
```

Comment:
The impact to local storage is important. However, we probably want more emphasis on the _network consumption_ ramifications, e.g., include some word like "download":

```suggestion
By default, DataStore downloads the entire contents of your cloud data source to your local device.
```